maestral_qt/utils.py

```diff
@@ -227,7 +227,7 @@ def get_masked_image(path, size=64, overlay_text=""):
     pm = QPixmap.fromImage(out_img)
     pm.setDevicePixelRatio(pr)
     size *= pr
-    pm = pm.scaled(size, size, Qt.KeepAspectRatio, Qt.SmoothTransformation)
+    pm = pm.scaled(int(size), int(size), Qt.KeepAspectRatio, Qt.SmoothTransformation)
 
```

Comment:
You perform the same conversion twice here. It's a bit more efficient to convert to integer once in the preceding line. I am only considering if using `round` instead of `int` might be better here (int always rounds down and difference may be significant when rounding the corners of an image).